metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -734,7 +691,9 @@ def from_api(  # noqa: C901
                 fields=view_fields,
                 upstream_views=list(
                     ProjectInclude(
-                        project=_BASE_PROJECT_NAME,
+                        project=_BASE_PROJECT_NAME
+                        if view_name not in view_project_map
```

Comment:
can change this to `view_project_map.get(view_name, _BASE_PROJECT_NAME)`

Reply:
done